apps/studio/src/utility.ts

```diff
@@ -0,0 +1,68 @@
+import rawLog from 'electron-log'
+import ORMConnection from './common/appdb/Connection'
+import platformInfo from './common/platform_info';
+import { connHandlers } from './handlers/connHandlers';
+import { exportHandlers } from './handlers/exportHandlers';
+import { generatorHandlers } from './handlers/generatorHandlers';
+import { Handlers } from './handlers/handlers';
+import { state } from './handlers/handlerState';
```

Comment:
I don't think you should export this constant `state` object, but rather instantiate it here in the utility file.

I don't see any other imports, so I think it's safe to just import the class, and create a new instance here in Utility.ts

Reply:
We do actually have to use it elsewhere. It is used in the connection handlers, the queryHandlers, the exportHandlers, and the generatorHandlers